core/chains/evm/gas/models.go

```diff
@@ -171,6 +171,33 @@ func (fee EvmFee) ValidDynamic() bool {
 	return fee.DynamicFeeCap != nil && fee.DynamicTipCap != nil
 }
 
+// Helper function to get the minimum of two values
+func min(a, b uint64) uint64 {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+// Utility function to calculate the highest gas price or fee for a given limit
+func getHighestPrice(maxGasCost, gasLimit uint64) uint64 {
+	return maxGasCost / gasLimit
```

Comment:
This helper adds a lot of complexity and indirection. What is the downside to doing this inline instead?